### PR TITLE
Add NamedValues to Nebula, Ion Storm and Molecular Cloud for Pedia

### PR DESCRIPTION
**Pull request by Oberlus**

### CODE REVIEW

**Comment by Oberlus** [comment]
With this PR applied to current master (e3d8204), consulting the Pedia for first time after starting FreeOrion needs half a minute maybe (in a high performance rig). Without the PR (no recompilation needed), consulting Pedia is instantaneous.

**Comment by geoffthemedio** [comment]
Check the client log. Probably there are a bunch (about 10 is the limit I think) errors about missing strings. If those are the issue, then usually after the limit is reached, searches should be faster as it stops logging the errors and has already populated the "ERROR: WHATEVER" string table.

**Comment by o01eg** [comment]
@geoffthemedio Could it be made into unit test about missing strings?

**Comment by geoffthemedio** [comment]
Probably, though the error log includes a stack trace that might be difficult to get into a unit test result...?

**Comment by Oberlus** [comment]
The problem is in the NamedRealLookup in two of the three FLD_NEBULA_*.
The NamedReal in the first nebula file gives no trouble, the other two create errors for a while in the log until resolved.
I'll try declaring the value in named_values.focs.txt and use the lookup in the three, or merging the three nebulae in a single FOCS file. ~~Not today.~~

**Comment by Oberlus** [comment]
Declaring the values in named_values.focs and using lookups in the three nebula focs files did the trick.

Ready for merge.

File: default/scripting/fields/FLD_NEBULA_1.focs.txt [hunk 1]
```diff
@@ -51,9 +51,12 @@ FieldType
             ]
             stackinggroup = "NEBULA_SHIP_EFFECTS"
             effects = [
-                SetSpeed value = max(5, Value - 20)
-                SetStealth value = Value + 20
-                SetDetection value = Value - 20
+                //SetSpeed value = max(5, Value - (NamedReal name = "FLD_NEBULA_SPEED_MALUS" value = 20))
+                //SetStealth value = Value + NamedReal name = "FLD_NEBULA_STEALTH_BONUS" value = 20
+                //SetDetection value = Value - NamedReal name = "FLD_NEBULA_DETECTION_MALUS" value = 20
```
Comment (geoffthemedio) [inline default/scripting/fields/FLD_NEBULA_1.focs.txt]:
```suggestion

```

Comment (Oberlus) [reply default/scripting/fields/FLD_NEBULA_1.focs.txt]: Ouch... I forgot to remove the commented code.

**Comment by agrrr3** [comment]
> The problem is in the NamedRealLookup in two of the three FLD_NEBULA_*. The NamedReal in the first nebula file gives no trouble, the other two create errors for a while in the log until resolved. 

@oberlus was that the message that it is waiting for a value ref not yet registered? and how did you open the pedia?

@o01eg is the python parsing running unconditionally (or is it lazy)?

**Comment by Oberlus** [comment]
> was that the message that it is waiting for a value ref not yet registered?

Yes

> and how did you open the pedia?

I started FO and clicked on Pedia. Was OK.
Then I clicked on FieldType (I was testing this PR) and FO freezed for some 30 seconds.

I guess if I wait a minute after starting FO before using Pedia, there won't be any delay.

**Comment by o01eg** [comment]
> @o01eg is the python parsing running unconditionally (or is it lazy)?

Python executes FOCS files uncoditionally.

**Pull request merged**